test/org/sagebionetworks/bridge/hibernate/HibernateAccountDaoTest.java

```diff
@@ -1984,17 +1672,6 @@ public void twoNoneOfGroupsQueryCorrect() throws Exception {
         assertTrue(query.contains(":notin2 not in elements(acct.dataGroups)"));
     }
     
-    public void authenticateAccountUnverifiedEmailSucceedsForLegacy() throws Exception {
-        HibernateAccount hibernateAccount = makeValidHibernateAccount(true, true);
-        hibernateAccount.setEmailVerified(false);
```

Comment:
Was this test and authenticateAccountUnverifiedPhoneSucceedsForLegacy() supposed to be deleted? It looks like they were intended to test an actual code path, but the Test annotation was mistakenly left off.